main.cpp

```diff
@@ -1,16 +1,33 @@
-#pragma once
-#include "src/raylib.h"
-#include "src/Scene.h"
-#include "src/SceneMainMenu.h"
-#include "src/SceneGame.h"
-#include "src/SceneEditor.h"
+#include "main.h"
 
+// Engine commands
+namespace command {
+bool _EXIT = false;
+}
+
+// Invoking functions - TODO(moliwa): maybe this should be moved?
+void invoke_process_input(void (*func)()) {
+  return func();
+}
 
 int main(int charc, char** argv) {
   InitWindow(screenWidth, screenHeight, "RayLib Space Game");
 
   SetTargetFPS(60);
 
+  // Init render control values here
+  bool dropdown_console = false;
+
+  // Console initialization
+  int x      = 10;
```

Comment:
Values that won't change during runtime should be marked as "const"

Reply:
Fixed